src/index.js

```diff
@@ -35,6 +36,10 @@ export default class SaucelabsConnector {
             retries:    WEB_DRIVER_CONFIGURATION_RETRIES,
             timeout:    WEB_DRIVER_CONFIGURATION_TIMEOUT
         });
+
+        this.options = {
+            showBrowserStartMessage: options.showBrowserStartMessage

```

Comment:
ok, then choose one of `disableConnectorLogs` or `disableConnectorLogging`, what do you like more) You can ask a technical writer for advice